sources/web/datalab/polymer/modules/gapi-manager/gapi-manager.ts

```diff
@@ -27,10 +28,112 @@ enum GapiScopes {
 const initialScopeString = 'profile email';
 
 /**
- * This file contains a collection of functions that interact with gapi.
+ * This module contains a collection of functions that interact with gapi.
  */
 class GapiManager {
 
+  public static drive = class {
```

Comment:
I like this refactoring. Should the Drive and BigQuery pieces each go into their own files?